content/cpp/concepts/strings/terms/length/length.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: '.length()'
+Description: 'Returns the length of the string it is called on.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The `.length()` method finds the length of the string it is called on, in terms of bytes. This is the same functionality as the [`.size()`](https://www.codecademy.com/resources/docs/python/strings/size) method.
```

Comment:
@Dusch4593 What do you suggest? In the documentation, they have the exact same definition. It was made because in speaking terminology some people say the size of something while others say the length of something. So the developers thought it made sense to just have both.

Reply:
Maybe something like "While the [`.size()`](..) method is similar and returns the same value, `.length()` is commonly used with characters and strings of words and sentences." (but less wordy than what I just typed lol)

I got the idea from reading [this excerpt](https://www.geeksforgeeks.org/5-different-methods-find-length-string-c/#:~:text=The%20size()%20function%20is,to%20make%20things%20more%20readable.) from a GeeksforGeeks article.

What do you think?

Reply:
@Dusch4593 We could add it. I'm thinking of a way to say it without being too wordy.

Reply:
How about this:
```suggestion
The .length() method returns the length of the given string in terms of bytes. It functions exactly like the [.size()](https://www.codecademy.com/resources/docs/python/strings/size) method but follows a more intuitive terminology, i.e., "the length of a string" rather than "the size of a string." However, both methods yield the same results.
```